libpod/runtime.go

```diff
@@ -281,10 +281,16 @@ func defaultRuntimeConfig() (RuntimeConfig, error) {
 	if err != nil {
 		return RuntimeConfig{}, err
 	}
+	graphRoot := storeOpts.GraphRoot
+	if graphRoot == "" {
+		graphRoot = "/var/lib/containers/storage"
```

Comment:
You should at least warn on this.

Reply:
Good point, added warning logs

Reply:
Concur, but could be talked into a debug instead.  Chocolate chip cookies would help sway me!  ;^)